Buildings/Controls/OBC/FDE/DOAS/SupplyFanController.mo

```diff
@@ -0,0 +1,224 @@
+within Buildings.Controls.OBC.FDE.DOAS;
+block SupplyFanController
+  "This block manages start, stop, status, and speed of the supply fan."
+
+ parameter Real minDDSPset(
+   min=0,
+   final unit="Pa",
+   final quantity="PressureDifference")=125
+    "Minimum down duct static pressure reset value"
+      annotation (Dialog(group="DDSP range"));
+ parameter Real maxDDSPset(
+   min=0,
+   final unit="Pa",
+   final quantity="PressureDifference")=500
+    "Maximum down duct static pressure reset value"
+      annotation (Dialog(group="DDSP range"));
+  parameter Real cvDDSPset(
+   min=0,
+   final unit="Pa",
+   final quantity="PressureDifference")=250
+    "Constant volume down duct static pressure set point";
+ parameter Real damSet(
+   min=0,
+   max=1,
+   final unit="1")=0.9
+   "DDSP terminal damper percent open set point";
+ parameter Boolean vvUnit = true
+   "Set true when unit serves variable volume system.";
+
+  // ---inputs---
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput occ
+    "True when occupied mode is active"
+    annotation (Placement(transformation(extent={{-142,60},{-102,100}}),
+       iconTransformation(extent={{-140,50},{-100,90}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput mostOpenDam
```

Comment:
so this input will not be needed when the unit is not serving variable volume system? In that case, then it can become an conditional instance, wihch is like:
```
Buildings.Controls.OBC.CDL.Interfaces.RealInput mostOpenDam if vvUnit
```